### DISCUSSION TITLE
Voice Chat Sessions

### DISCUSSION THREAD

**Discussion by torrentula993** [lencx/ChatGPT#410] [Ideas]
Using the standard OpenAI ChatGPT web interface, occasionally I've found myself waiting to hear the model respond using voice. It seems such a natural progression for these interactions, especially considering we're already using a natural language communication paradigm. 

Obviously there are two directions voice be made part of the interface -- 1) prompts TO and 2) responses FROM the model. If voice were introduced it might be easier and make sense to implement response voiceovers first. Further, composing prompts can require an iterative trial-and-error editing process which voice would complicate. 

Curious to hear people's thoughts.

**Comment by Ethkuil** [comment]
I will use voice input iff I would like to have a oral "chat". I will stick to keyboard if I want to just let ChatGPT helps me do something.

And a similar issue: #113.

**Comment by CassinianSoftware** [comment]
@torrentula993 Yes, I was thinking along the very same lines. TTS that gives ChatGPT a voice would be nice, assuming it was a decent-quality TTS. Done right, it could add a layer of "humanness" to the chat. The other way around, however, could be nice in some situations. But as @Ethkuil points out, the keyboard is best when I need actual assistance with something. And as you said, composing prompts can be an iterative process. But combining TTS and STT could be fun for casual chats. Of course, then we'd need a window with an animated face that could lip-sync with the TTS!

**Comment by bhaskatripathi** [comment]
I made this:
<iframe
	src="https://bhaskartripathi-text2videolisa.hf.space"
	frameborder="0"
	width="850"
	height="450"
></iframe>

**Comment by bhaskatripathi** [comment]
https://github.com/bhaskatripathi/AskLisa
